activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -222,12 +222,32 @@ def includes!(*args) # :nodoc:
       self
     end
 
-    # Forces eager loading by performing a LEFT OUTER JOIN on +args+:
+    # Specify associations +args+ to be eager loaded using a LEFT OUTER JOIN.
```

Comment:
```suggestion
    # Specify associations +args+ to be eager loaded using a <tt>LEFT OUTER JOIN</tt>.
```

I feel there is some duplication here in the next sentence too 🤔

Reply:
Good point. I'll reword it.